ui/src/widgets/ui-text-input/UITextInput.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <v-tooltip :text="tooltip" :disabled="!tooltip?.length" location="bottom">
+    <v-tooltip :key="key" :text="tooltip" :disabled="!tooltip?.length" location="bottom">
```

Comment:
not entirely sure this is required (I forget when/why I felt the need to add the key)